playbooks/common/openshift-cluster/initialize_facts.yml

```diff
@@ -8,3 +8,5 @@
       role: common
       local_facts:
         hostname: "{{ openshift_hostname | default(None) }}"
+  - set_fact:
+      openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
```

Comment:
Hoping to apply this approach to fix additional issues such as:

https://bugzilla.redhat.com/show_bug.cgi?id=1347303

https://bugzilla.redhat.com/show_bug.cgi?id=1345804